examples/meta/src/evaluation/cross_validation_multiple_kernel_learning_weights_storage.sg

```diff
@@ -46,8 +46,9 @@ real stddev = result.get_real("std_dev")
 #![get_results]
 
 #![get_fold_machine]
-CrossValidationStorage obs = mkl_obs.get_observation(0)
-CrossValidationFoldStorage fold = obs.get_fold(0)
+ObservedValue obs_v = mkl_obs.get_observation(0)
+CrossValidationStorage stor = obs_v.get_CV_storage("value")
+CrossValidationFoldStorage fold = stor.get_fold(0)
```

Comment:
IMO, we souldnt add this many new types to the API. Instead can't we use `put/get` to retreive this info?

`mkl_obs.get_observation(0)..get("value").get("fold", 0)`
Even better if the `get("value") wouldnt be there but the values would be directly accessible from the value
`mkl_obs.get_observation(0).get("fold", 0)`

@lisitsyn we need a `get` for array elements, similar to the `add` we have. Maybe we can just do this as above, with the additional `int` parameter in get?

Reply:
`get("fold", 0)` would be super useful indeed.

Reply:
Yep let’s add it. I’ll be travelling tomorrow and will have a look.

Reply:
Get for arrays is now availavle